backend/tracim_backend/views/contents_api/file_controller.py

```diff
@@ -200,7 +219,11 @@ def preview_pdf_full(self, context, request: TracimRequest, hapic_data=None):  #
             content_type=CONTENT_TYPES.Any_SLUG
         )
         pdf_preview_path = api.get_full_pdf_preview_path(content.revision_id)
-        return FileResponse(pdf_preview_path)
+        response = FileResponse(pdf_preview_path)
+        # INFO - G.M - 2018-09-05 - Support for force download param
+        if hapic_data.query.force_download:
+            response.content_disposition = 'attachment'
```

Comment:
@inkhey you must factorize this kind of code!

Two ways of doing this:
- add a parameter force_download in `FileResponse` class
- implement and use a decorator

You must factorize this kind of code

Reply:
done.